### ISSUE TITLE
Crash when determining user interface size

### ISSUE THREAD

**Issue by RampantPanda** [damonlynch/rapid-photo-downloader#67]
Hi,
Like the title says. I get the "A problem occurred" screen.

Attaching the bug report file:
[rpd-bug-report-20220227.tar.gz](https://github.com/damonlynch/rapid-photo-downloader/files/8148699/rpd-bug-report-20220227.tar.gz)

Here's the error message:
```
Traceback (most recent call last):
  File "/usr/bin/rapid-photo-downloader", line 33, in <module>
    sys.exit(load_entry_point('rapid-photo-downloader==0.9.29', 'gui_scripts', 'rapid-photo-downloader')())
  File "/usr/lib/python3.10/site-packages/raphodo/rapid.py", line 7881, in main
    rw = RapidWindow(
  File "/usr/lib/python3.10/site-packages/raphodo/rapid.py", line 476, in __init__
    self.readWindowSettings(app)
  File "/usr/lib/python3.10/site-packages/raphodo/rapid.py", line 1514, in readWindowSettings
    self.resizeAndMoveMainWindow()
  File "/usr/lib/python3.10/site-packages/raphodo/rapid.py", line 1482, in resizeAndMoveMainWindow
    was_valid, validatedSize = validateWindowSizeLimit(available.size(), size)
  File "/usr/lib/python3.10/site-packages/raphodo/viewutils.py", line 801, in validateWindowSizeLimit
    width_valid = desired.width() <= available.width()
AttributeError: 'NoneType' object has no attribute 'width'
```

**Comment by damonlynch** [comment]
What platform are you running this on? What if anything changed in your system between the first successful run of 0.9.29, and these subsequent runs in which you have seen the crash?

**Comment by vitberget** [comment]
I had the same problem. When i moved `.config/Rapid Photo Downloader/Rapid Photo Downloader.conf`it started up as normal. After looking a little in the code, I saw that  [MainWindow] windowSize was missing, When adding windowSize=@Size(958 1182) it started with the old conf-file.

TL;DR removing [MainWindow]  windowSize=@Size(958 1182) will crash the application

**Comment by damonlynch** [comment]
Excellent sleuthing @vitberget thanks 👍
